docs/index.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img width="320" height="192" src="https://raw.githubusercontent.com/encode/starlette/master/docs/starlette.png" alt='starlette'>
+  <img width="100%" src="https://user-images.githubusercontent.com/11155743/55277958-02552b00-5317-11e9-8780-ef9874d137b2.jpg" alt='starlette'>
```

Comment:
Rather than changing the location here, I'm happy for the logo *itsself* to replace `docs/starlette.png` for this pull request.

Reply:
Oh, I understand.
I push new logo to `docs/starlette.png`